db.go

```diff
@@ -138,6 +139,19 @@ func newDBMetrics(db *DB, r prometheus.Registerer) *dbMetrics {
 		defer db.mtx.RUnlock()
 		return float64(len(db.blocks))
 	})
+	m.symbolTableSize = prometheus.NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "prometheus_tsdb_symbol_table_size",
```

Comment:
See: https://prometheus.io/docs/practices/naming/#metric-names

Should be `prometheus_tsdb_symbol_table_size_bytes`